### PR TITLE
add testgrid Gardener k8s release version 1.13

### PR DESCRIPTION
**Pull request by OlegLoewen**

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -3214,32 +3214,32 @@ test_groups:
   num_failures_to_alert: 1
 
 # Gardener Test Groups
-- name: ci-gardener-e2e-conformance-aws-v1.11
-  gcs_prefix: k8s-conformance-gardener/ci-gardener-e2e-conformance-aws-v1.11
+- name: ci-gardener-e2e-conformance-aws-v1.13
```
Comment (BenTheElder) [inline testgrid/config.yaml]: will note to come back and document later but I know we have major time-zone clashes so will go ahead and get this in - to confirm these are 1.13 stable, as in 1.13.2, or 1.13.X under development (IE build from release-1.13 at HEAD) ?

Comment (OlegLoewen) [reply testgrid/config.yaml]: Hi @BenTheElder, it's 1.13.2 version.

Comment (BenTheElder) [reply testgrid/config.yaml]: thanks for clarifying, at some point I want to further organize how we document this :+1:

**Pull request merged**